cmd/tools/sql/main.go

```diff
@@ -29,6 +29,7 @@ import (
 
 	_ "go.temporal.io/server/common/persistence/sql/sqlplugin/mysql"      // needed to load mysql plugin
 	_ "go.temporal.io/server/common/persistence/sql/sqlplugin/postgresql" // needed to load postgresql plugin
+	_ "go.temporal.io/server/common/persistence/sql/sqlplugin/sqlite"     // needed to load sqlite plugin
```

Comment:
this is probably unnecessary

Reply:
I assume without it you can't set up SQLite in the file.